### PR TITLE
Minor: Error messages + descriptions

### PR DESCRIPTION
**Pull request by paquettealice**
- Added some formatting to error message printouts as they were a little difficult to decode with the way everything was concatenated on the same line
- Made a few typo fixes or updates to descriptions

### CODE REVIEW

File: projects/ngrx-auto-entity/src/lib/operators.ts [hunk 1]
```diff
@@ -74,18 +74,21 @@ export const handleError = <TModel, TErrorAction>(
   if (error.err instanceof TypeError) {
     const serviceName = `${pascalCase(error.info.modelName)}Service`;
     console.error(
-      `[NGRX-AE] ! NgRxAutoEntityService Error: Unable to locate load method in the ${serviceName}`,
+      `[NGRX-AE] ! NgRxAutoEntityService Error: Unable to locate load method in the ${serviceName}`,,
```
Comment (tonivj5) [inline projects/ngrx-auto-entity/src/lib/operators.ts]: double `,`.

I think it produces a SyntaxError :thinking:

Comment (paquettealice) [reply projects/ngrx-auto-entity/src/lib/operators.ts]: Woops, copy paste mishap. Should be fixed now.

**Comment by paquettealice** [comment]
Not sure why the review got dismissed (perhaps because I added a commit? Still unfamiliar with the flow), but regardless, we'll be merging these changes with version 0.3.0 (which will add correlation ids).

**Comment by jrista** [comment]
> Not sure why the review got dismissed (perhaps because I added a commit? Still unfamiliar with the flow), but regardless, we'll be merging these changes with version 0.3.0 (which will add correlation ids).

Yeah, I currently have it configured to dismiss prior reviews when updates are pushed. The PR is still open, but the review was dismissed.

**Comment by jrista** [comment]
Thanks for the review, @tonivj5!

**Comment by jrista** [comment]
Applied in feat/0.3.0 branch.

**Pull request closed without merging**